packages/angular/cli/upgrade/version.ts

```diff
@@ -70,9 +95,9 @@ export class Version {
       return;
     }
 
-    if (!angularVersion.isGreaterThanOrEqualTo(new SemVer('5.0.0'))) {
+    if (!angularVersion.satisfies(supportedAngularSemver)) {
       console.error(terminal.bold(terminal.red(tags.stripIndents`
-          This version of CLI is only compatible with Angular version 5.0.0 or higher.
+          This version of CLI is only compatible with Angular versions ${supportedAngularSemver}.
```

Comment:
I think this should also tell the user what versions they currently have.  Thoughts?

Reply:
I think that's a good idea, added.